CMakeLists.txt

```diff
@@ -2419,7 +2419,8 @@ endif()
 # Packaging
 set(CPACK_PACKAGE_VENDOR "Mixxx Project")
 set(CPACK_PACKAGE_CONTACT "RJ Skerry-Ryan <rryan@mixxx.org>")
-set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/cmake/cpack_package_description.txt")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Mixxx, powerfull features for all DJs")
```

Comment:
```suggestion
set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Mixxx DJ software")
```

Reply:
No need for embellishing language here, just a plain description is more helpful IMO